### PR TITLE
MES-6297 Migrate Debrief Page

### PR DESCRIPTION
**Pull request by Cullima**
## Description
- Creates a unified, category agnostic debrief page.

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

## Screenshots (optional)
Pass:
image
Fail:
image
Terminate:
image

### CODE REVIEW

File: src/app/pages/debrief/debrief.page.ts [hunk 1]
```diff
@@ -0,0 +1,200 @@
+import { NavController, Platform } from '@ionic/angular';
+import { PracticeableBasePageComponent } from '@shared/classes/practiceable-base-page';
+import { select, Store } from '@ngrx/store';
+import { StoreModel } from '@shared/models/store.model';
+import { AuthenticationProvider } from '@providers/authentication/authentication';
+import { getCurrentTest, getJournalData } from '@store/tests/tests.selector';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (tomcrawleyy) [inline src/app/pages/debrief/debrief.page.ts]: could this be removed?

**Review by tomcrawleyy — APPROVED** [review]

File: src/app/pages/debrief/debrief.page.ts (hunk 1 above)
Comment (Cullima) [reply src/app/pages/debrief/debrief.page.ts]: yes, removed

File: src/app/pages/confirm-test-details/confirm-test-details.page.ts [hunk 2]
```diff
@@ -13,6 +13,7 @@ export class ConfirmTestDetailsPage implements OnInit {
   constructor(
     private navController: NavController,
```
Comment (andrewsetterfield) [inline src/app/pages/confirm-test-details/confirm-test-details.page.ts:14]: can u remove navcontroller if your not using it any more

Comment (Cullima) [reply src/app/pages/confirm-test-details/confirm-test-details.page.ts:14]: not really part of this merge but will do. will be lots of skeleton pages suing it atm they will be over-written as migrated

**Review by andrewsetterfield — APPROVED** [review]

**Pull request merged**